common/device-common.mk

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ifneq (,$(filter $(TARGET_PRODUCT),hikey960_tv hikey_tv))
+ifneq (,$(filter $(TARGET_PRODUCT),plus2e_tv))
 # Setup TV Build
```

Comment:
There is plans to reuse this file for other boards/SOCs, please use another flag to distinguish between product type

Reply:
Done.